src/Jira/Issues/Walker.php

```diff
@@ -260,6 +263,7 @@ public function valid()
 	 * @return void Any returned value is ignored.
 	 * @link   http://php.net/manual/en/iterator.rewind.php
 	 */
+    #[\ReturnTypeWillChange]
```

Comment:
This kind of stuff would result in an exception/fatal error on PHP versions prior to PHP 8.0. Definitely remove.

Reply:
We'll see once the tests are fixed.